docs/source/configuration/environmentvariables.md

```diff
@@ -89,3 +89,77 @@ It displays the errors of the non-compliant customizations (in server console) i
 #### Internationalization errors (i18n)
 
 It will enable the log of missing i18n messages (in console).
+
+## Add add-ons via the `ADDONS` environment variable
```

Comment:
"Add" should be replaced with "Use"

Reply:
Fixed in ea8b7fa88df8c1981a2ef0fbf4bcac0092799923